harness/determined/cli/checkpoint.py

```diff
@@ -168,6 +182,20 @@ def describe(args: Namespace) -> None:
             "describe checkpoint",
             [Arg("uuid", type=str, help="checkpoint uuid to describe")],
         ),
+        Cmd(
+            "delete",
+            delete_checkpoints,
+            "delete checkpoints",
+            [
+                Arg("checkpoints_uuids", help="comma-separated list of checkpoints to delete"),
```

Comment:
just thinking, what is better `det c delete 1,2,3` or `det c delete 1 2 3`? thoughts?

Reply:
I think 1,2,3 looks more like a list than 1 2 3. But I don't think I have strong preference. What do most cli's do? I saw that in experiment.py (describe experiments) we use commas.

Reply:
let's just do commas then if experiment.py does that